src/main/java/org/example/ClassSpells.java

```diff
@@ -0,0 +1,28 @@
+package org.example;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class ClassSpells {
+    public final String userURL = "https://www.dnd5eapi.co/api/classes/:index/levels/1/spells";
+    private final HttpClient client = HttpClient.newHttpClient();
+
+    public HttpResponse<String> getClassSpells(String index) {
+        HttpRequest request;
+        try {
+            request = HttpRequest.newBuilder(new URI(userURL.replace(":index", index))).GET().build();
+            HttpResponse<String>response = client.send(request,HttpResponse.BodyHandlers.ofString());
+            return response;
+        } catch (URISyntaxException e) {
```

Comment:
all those catch blocks have the same response. Please create separate responses or just a single catch block

Reply:
IOException | InterruptedException

Reply:
Done